File: src/main/java/com/radar/prospection/ProspectionRadarApplication.java
package com.radar.prospection;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling
public class ProspectionRadarApplication {

    public static void main(String[] args) {
        SpringApplication.run(ProspectionRadarApplication.class, args);
    }
}

File: backend/src/main/kotlin/com/radar/prospection/ProspectionRadarApplication.kt
package com.radar.prospection

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
class ProspectionRadarApplication

fun main(args: Array<String>) {
    runApplication<ProspectionRadarApplication>(*args)
}
